treeherder/model/models.py

```diff
@@ -1128,12 +1155,17 @@ class ClassifiedFailure(models.Model):
     id = BigAutoField(primary_key=True)
     failure_lines = models.ManyToManyField(FailureLine, through='FailureMatch',
                                            related_name='classified_failures')
+    text_log_errors = models.ManyToManyField("TextLogError", through='TextLogErrorMatch',
+                                             related_name='classified_failures')
     # Note that we use a bug number of 0 as a sentinel value to indicate lines that
     # are not actually symptomatic of a real bug, but are still possible to autoclassify
     bug_number = models.PositiveIntegerField(blank=True, null=True, unique=True)
     created = models.DateTimeField(auto_now_add=True)
     modified = models.DateTimeField(auto_now=True)
 
+    def __str__(self):
+        return "id:%s bug_number:%s" % (self.id, self.bug_number)
```

Comment:
I think I would prefer if you'd use the standard format for __str__ we use elsewhere in this file (ditto for everything else), which generally means skipping the id and just showing the fields that distinguish this object. The inconsistency is jarring.